src/tcgwars/logic/impl/gen3/DeltaSpecies.groovy

```diff
@@ -2887,14 +2887,14 @@ public enum DeltaSpecies implements LogicCardInfo {
           def toDiscard = my.hand.getExcludedList(thisCard).select(count:1, "Discard a card to discard.")
           toDiscard.discard()
 
-          if (opp.hand.size() > my.hand.size()) {
+          if (opp.hand.size() > my.hand.size()-1) {
```

Comment:
Is this `-1` technically this card? If so
`my.hand.getExcludedList(thisCard).size()` might be a little bit more verbose, but has more clarity.